azurerm/data_source_kubernetes_cluster.go

```diff
@@ -308,6 +308,11 @@ func dataSourceArmKubernetesCluster() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+
+						"load_balancer_sku": {
```

Comment:
could we add this new field [to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/d/kubernetes_cluster.html.markdown#L35)?

Reply:
✅